packages/expo-av/src/GlobalPlatformEmitter.ts

```diff
@@ -0,0 +1,17 @@
+import RCTDeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter';
```

Comment:
Can you open up a PR to move this module to expo-react-native-adapter and export it through expo-core?

Reply:
#3368